lib/ansible/module_utils/network/vyos/config/lag_interfaces/lag_interfaces.py

```diff
@@ -0,0 +1,381 @@
+
+        if 'interval' in arp_monitor:
+            commands.append(
+                self._compute_command(key, 'interval', str(arp_monitor['interval']))
+            )
+        if diff_targets:
+            for target in diff_targets:
+                commands.append(
+                    self._compute_commands(key, 'target', target)
+                )
+        return commands
+
+    def _delete_bond_members(self, have):
+        commands = []
+        for member in have['members']:
+            commands.append(
+                self._compute_command(
+                    member['member'], 'bond-group', have['name'], False, 'ethernet'
+                )
+            )
+        return commands
+
+    def _update_arp_monitor(self, key, want, have):
+        commands = []
+        want_arp_target = []
+        have_arp_target = []
+        want_arp_monitor = want.get(key) or {}
+        have_arp_monitor = have.get(key) or {}
+        del_cmd = 'delete interface bonding ' + have['name']
+
+        if want_arp_monitor and 'target' in want_arp_monitor:
+            want_arp_target = want_arp_monitor['target']
+
+        if have_arp_monitor and 'target' in have_arp_monitor:
+            have_arp_target = have_arp_monitor['target']
+
+        if 'interval' in have_arp_monitor and not want_arp_monitor:
+            commands.append(del_cmd + ' ' + key + ' interval')
+        if 'target' in have_arp_monitor:
+            target_diff = list_diff_have_only(want_arp_target, have_arp_target)
+            if target_diff:
+                for target in target_diff:
+                    commands.append(del_cmd + ' ' + key + ' target ' + target)
+
+        return commands
+
+    def _update_bond_members(self, key, want, have):
+        commands = []
+        want_members = want.get(key) or []
+        have_members = have.get(key) or []
+
+        members_diff = list_diff_have_only(want_members, have_members)
+        if members_diff:
+            for member in members_diff:
+                commands.append(
+                    self._compute_command(
+                        member[key], 'bond-group', have['name'], False, 'ethernet'
+                    )
+                )
+        return commands
+
+    def _get_arp_monitor_target_diff(self, want_list, have_list, dict_name, lst):
+        want_arp_target = []
+        have_arp_target = []
+
+        want_arp_monitor = want_list.get(dict_name) or {}
+        if want_arp_monitor and lst in want_arp_monitor:
+            want_arp_target = want_arp_monitor[lst]
+
+        if not have_list:
+            diff = want_arp_target
+        else:
+            have_arp_monitor = have_list.get(dict_name) or {}
+            if have_arp_monitor and lst in have_arp_monitor:
+                have_arp_target = have_arp_monitor[lst]
+
+            diff = list_diff_want_only(want_arp_target, have_arp_target)
+        return diff
+
+    def _compute_command(self, name, attrib, value=None, set_flag=True, type='bonding'):
```

Comment:
`type` can be determined from name of the interface - you can refer to `get_interface_type` which is already added in utils.

Reply:
Same